Count,Message,Traceback,Nodes
1,Object of type _GenericAlias is not JSON serializable,"  File ""/opt/anaconda3/envs/xn/lib/python3.11/site-packages/locust/user/task.py"", line 340, in run
    self.execute_next_task()
  File ""/opt/anaconda3/envs/xn/lib/python3.11/site-packages/locust/user/task.py"", line 373, in execute_next_task
    self.execute_task(self._task_queue.popleft())
  File ""/opt/anaconda3/envs/xn/lib/python3.11/site-packages/locust/user/task.py"", line 490, in execute_task
    task(self.user)
  File ""/Users/jianruitian/Documents/GitHub/bce-qianfan-sdk/python/qianfan/dataset/stress_test/qianfan_llm_load.py"", line 699, in mytask
    self.client.qianfan_request(
  File ""/Users/jianruitian/Documents/GitHub/bce-qianfan-sdk/python/qianfan/dataset/stress_test/qianfan_llm_load.py"", line 134, in qianfan_request
    request_meta = self._request_internal(context=context, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ""/Users/jianruitian/Documents/GitHub/bce-qianfan-sdk/python/qianfan/dataset/stress_test/qianfan_llm_load.py"", line 353, in _request_internal
    res = json.dumps(res,ensure_ascii=False)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ""/opt/anaconda3/envs/xn/lib/python3.11/json/__init__.py"", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File ""/opt/anaconda3/envs/xn/lib/python3.11/json/encoder.py"", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ""/opt/anaconda3/envs/xn/lib/python3.11/json/encoder.py"", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File ""/opt/anaconda3/envs/xn/lib/python3.11/json/encoder.py"", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
",MacBook-Pro.local_ece88b0df17e4d969b6b618a05667681
